.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature Request, needs triage
```

Comment:
We've been using 'feature' label, but mostly 'enhancement' label.

It hasn't been particularly useful to distinguish between feature (24) and enhancement (192) though.

I propose we label all the existing 'features' with 'enhancement', then get rid of 'feature' label, get rid of this new 'Feature request' label and make the template use 'enhancement' and 'needs triage'.